modules/local/orthofinder_prep.nf

```diff
@@ -0,0 +1,58 @@
+process ORTHOFINDER_PREP {
+    tag "Prepping data for OrthoFinder"
+    label 'process_low'
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/orthofinder:2.5.4--hdfd78af_0' :
+        'quay.io/biocontainers/orthofinder:2.5.4--hdfd78af_0' }"
+
+    //publishDir(
+    //    path: "${params.outdir}/orthofinder/",
+    //    mode: 'copy',
+    //    saveAs: { fn -> fn.substring(fn.lastIndexOf('/')+1) },
+    //)
+    
+    input:
+    val fasta_dir
+    val mcl_test
+    
+    output:
+    path "*.dmnd", emit: dmd
+    path "*.fa", emit: fa
+    path "SequenceIDs.txt", emit: seqIDs
+    path "SpeciesIDs.txt", emit: sppIDs
+    path "versions.yml" , emit: versions
+
+    script:
+    def testing_mcl = mcl_test.equals('true') ? "${mcl_test}" : "false"
+    """
+    # Prepare the bespoke orthofinder directory structure and stripped down files
+    # spit the output commands to a tmp file - we don't care about this
+    orthofinder \\
+        -f $fasta_dir \\
+        -t ${task.cpus} \\
+        -op > tmp
+        
+    # Create the publishdir if it's not made yet
+    mkdir -p ../../../${params.outdir}/orthofinder/
```

Comment:
crap, you're probably right. And will probably break depending on how it pulls data down from S3. This is something I had a hard time figuring out how to deal with to be honest.

I think these orthofinder modules are where I came up with the hacky-est solutions since I was the least familier with nextflow at some point. This can definitely be improved.